src/components/form/examples/dynamic-form.tsx

```diff
@@ -45,7 +45,14 @@ export class DynamicFormExample {
 
     public render() {
         return [
-            <textarea onChange={this.handleTextChange}>{this.text}</textarea>,
```

Comment:
in the previous version, one could typ inside the textarea and see the results live in the form below. We should make sure that the same thing happens when using the code-editor